cost/kubecost/cluster/README.md

```diff
@@ -0,0 +1,27 @@
+# Kubecost Cluster Sizing Recommendations cust
+
+## What it does
+
+The Kubecost Cluster Sizing Recommendations cust policy utilizes a Kubecost Host API endpoint to provide three different types of recommendations to resize clusters stored in it: single, multi and optimal.
+
+## Prerequisites
+
+This policy does not use credentials to access resources in a Kubecost Host API. In order to apply this policy you must enable external access exposing access to port 9090 on your pod following this [example](https://docs.kubecost.com/install-and-configure/install/ingress-examples).
+
+## Functional Details
+
+- The policy receives recommendations to resize clusters managed by Kubecost.
+- The policy calls a Kubecost Host API endpoint.
+
+### Input Parameters
+
+- *Email addresses* - A list of email addresses to notify.
+- *Kubecost Host or IP address* - The host or IP Address where Kubecost is running.
+- *Minimum node count* - Minimum allowed node count in the cluster.
+- *Number of days to analyze* - Historical range of time in days to analyze.
+- *Target utilization (%)* - Desirable target utilization in percent.
+- *Recommendation Type* - Type of existing recommendation in the cluster.
```

Comment:
Let's rename to "Recommendation Strategy" 
- Single. Provides cluster rightsizing recommendation for a uniform node type.
- Multi. Provides cluster rightsizing recommendation allowing different node types.
- Optimal. Provide cluster rightsizing recommendation with most savings, could be single or multi.

Reply:
Updated to "recommendation strategy" and added mentioned bullet points.